src/wallet/wallet_rpc_server_commands_defs.h

```diff
@@ -47,7 +47,7 @@
 // advance which version they will stop working with
 // Don't go over 32767 for any of these
 #define WALLET_RPC_VERSION_MAJOR 1
-#define WALLET_RPC_VERSION_MINOR 23
+#define WALLET_RPC_VERSION_MINOR 24
```

Comment:
Does this need to be bumped? If the client doesn't send user/pass, I think this just silently provides `boost::none` ?

Reply:
It might not be necessary technically but we usually always do it when editing that file.